airtime_mvc/application/models/Preference.php

```diff
@@ -498,7 +498,12 @@ public static function SetStationDescription($description)
 
     public static function GetStationDescription()
     {
-        return self::getValue("description");
+        $description = self::getValue("description");
+        if (!empty($description)) {
+            return $description;
+        } else {
+            return "Powered by Airtime Pro";
```

Comment:
Yeah. "Airtime Pro" should be one of the branding constants too, with that interpolated into it with sprintf.